File: Java/Piramidka.java
import java.util.Scanner;

public class Main 
{
    public static void printPyramid(int n, String s) 
    {
        int mid = n / 2;
        for (int i = mid; i >= 0; i--) 
        {
            String left = ".".repeat(i);
            String middle = s.substring(i, n - i);
            System.out.println(left + middle + left);
        }
    }

    public static void main(String[] args) 
    {
        Scanner input = new Scanner(System.in);
        int n = input.nextInt();
        String s = input.next();
        printPyramid(n, s);
        input.close();
    }
}

File: Python/Piramidka.py
def print_pyramid(n, s):
    mid = n // 2
    for i in range(mid, -1, -1):
        left = '.' * i
        middle = s[i:n - i]
        right = '.' * i
        print(left + middle + right)

n = int(input())
s = input()
print_pyramid(n, s)